Source/SwiftLintCore/Models/Linter.swift

```diff
@@ -265,7 +265,7 @@ public struct CollectedLinter {
     }
 
     private func cachedStyleViolations(benchmark: Bool = false) -> ([StyleViolation], [(id: String, time: Double)])? {
-        let start: Date! = benchmark ? Date() : nil
+        let start: Date! = benchmark ? Date() : nil // swiftlint:disable:this implicitly_unwrapped_optional
```

Comment:
That ought not to be a big deal:
```suggestion
        let start = Date()
```